src/workerd/util/sqlite.c++

```diff
@@ -150,8 +154,9 @@ class SqliteCallScope {
 // the return value of sqlite3_errmsg() or a string literal containing a similarly
 // application-approriate error message. A reference called `regulator` must be in-scope.
 // sqliteErrorCode is a kj::Maybe<int> and represents the error code from sqlite.
-#define SQLITE_REQUIRE(condition, sqliteErrorCode, errorMessage, ...)                              \
+#define SQLITE_REQUIRE(condition, sqliteErrorCode, errorMessage, maybeException, ...)              \
```

Comment:
Adding this `maybeException` param to the macro seems to add a lot of noise to the places that use the macro directly, none of which actually pass an error code and thus they cannot actually trigger critical errors.

Maybe we should put the `handleCriticalError()` call inside of `SQLITE_CALL`/`SQLITE_CALL_FAILED` after all. Plus that way there's always a `SqliteCallScope`...